src/ResourceGraph/ResourceGraph/Az.ResourceGraph.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.7.5'
```

Comment:
This is a preview module so that it change does't reflect in the overall change log. Am I right?

Reply:
you're right.